metadata-foundationdb/metadata-foundationdb.h

```diff
@@ -28,7 +25,7 @@ void metadata_persist(FDBDatabase *database, const Metadata *meta_data);
  * \return the MetaData stored in FoundationDB
  * \retval NULL if the MetaData could not be found or if there was an error unpacking
  */
-Metadata *metadata_get(FDBDatabase *database);
+Metadata *metadata_get();
```

Comment:
you renamed that to `load_metadata()` above

Reply:
good catch, thanks